### PR TITLE
feat(FasterKv):  Add FasterKv support

### PR DESCRIPTION
**Pull request by InCerryGit**
FasterKv is a hybrid memory and disk Kv Store, so it can support much larger data storage than memory. 
* Add FasterKv caching  support
* It only supports simple read and write operations, so for the time being there are more features not yet implemented（like, perfix query, expiration，etc.）

### CODE REVIEW

File: src/EasyCaching.FasterKv/Configurations/FasterKvOptionsExtension.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+using EasyCaching.Core.Serialization;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace EasyCaching.FasterKv.Configurations
+{
+    /// <summary>
+    /// LiteDB options extension.
```
Comment (catcherwong) [inline src/EasyCaching.FasterKv/Configurations/FasterKvOptionsExtension.cs:13]: Wrong summary

File: src/EasyCaching.FasterKv/Configurations/FasterKvOptionsExtension.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        /// </summary>
+        private readonly string _name;
+
+        /// <summary>
+        /// The configure.
+        /// </summary>
+        private readonly Action<FasterKvCachingOptions> _configure;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:EasyCaching.LiteDB.LiteDBOptionsExtension"/> class.
```
Comment (catcherwong) [inline src/EasyCaching.FasterKv/Configurations/FasterKvOptionsExtension.cs:28]: 😄

**Review by catcherwong — CHANGES_REQUESTED** [review]

File: src/EasyCaching.FasterKv/Configurations/FasterKvOptionsExtension.cs (hunk 1 above)
Comment (InCerryGit) [reply src/EasyCaching.FasterKv/Configurations/FasterKvOptionsExtension.cs:13]: OK

File: src/EasyCaching.FasterKv/Configurations/FasterKvOptionsExtension.cs (hunk 2 above)
Comment (InCerryGit) [reply src/EasyCaching.FasterKv/Configurations/FasterKvOptionsExtension.cs]: fixed

**Pull request merged**